### ISSUE TITLE
Unexpected behavior missing newline after "(".

### ISSUE THREAD

**Issue by kepocnhh** [pinterest/ktlint#1237]
I have upgraded from version `0.36.0` to `0.42.1`. Now the ktlint unexpectedly for me reacts to missing newline after "(".

See my code for example:
```kotlin
addView(TextView(context).also {
    it.layoutParams = LinearLayout.LayoutParams(
        width,
        height
    )
    it.gravity = gravity
    it.setTextColor(color)
    it.textSize = size
    it.text = text
})
```

## Expected Behavior
The ktlint requires me to wrap to a new line only if the parameter list does not fit on one line. Currently, only one parameter is passed to the function `addView`. The code for this parameter is carried over according to the rules.

## Observed Behavior
The ktlint requires me to wrap `TextView` to a new line. Like:
```kotlin
addView(
    TextView(context).also {
        ...
    }
)
```

I've used code constructs like this before. This was useful when you need to call a lot of similar methods. Like:
```kotlin
addView(View(context).also {
    ...
})
addView(View(context).also {
    ...
})
addView(View(context).also {
    ...
})
...
```
Otherwise, this approach greatly increases the height of the code. Especially if there is a lot of similar code inside such code. Like:
```kotlin
addView(ViewGroup(context).also { root ->
    root.addView(View(context).also {
        ...
    })
    ...
})
```

## Your Environment
* Version of ktlint used: 0.42.1
* Custom Gradle task:
```kotlin
val kotlinLint: Configuration by configurations.creating
dependencies {
    kotlinLint("com.pinterest:ktlint:0.42.1") {
        attributes {
            attribute(Bundling.BUNDLING_ATTRIBUTE, objects.named(Bundling.EXTERNAL))
        }
    }
}
task<JavaExec>("verifyCodeStyle") {
    classpath = kotlinLint
    main = "com.pinterest.ktlint.Main"
    args(
        "app/src/main/java/**/*.kt",
        "--reporter=html,output=$outputPath"
    )
}
```
* Version of Gradle used (if applicable): 7.0.2
I use the ktlint only from the console, without binding to any IDE. The main purpose is CI.

**Comment by dazza5000** [comment]
@paul-dingemans I am hitting this issue with ktlint ` "0.45.2"`` - ktlint is hitting on this during check, but the intellij formatter is not formatting it correctly.

This is an example block

```
                        tabTitles.forEachIndexed { index, title ->
                            Tab(selected = tabIndex == index,
                                onClick = { tabIndex = index },
                                text = { Text(text = title) })
                        }
```

This line is throwing the following
`                            Tab(selected = tabIndex == index,`


/source/source/well/consumer-android-ui-2/app/src/debug/java/co/well/wellapp/DesignSystem.kt:99:33 Missing newline after "("
